contrib_bots/lib/readme-github-comment-bot.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+This is the documentation for how to set up and run the github comment bot. (`githubcomment.py`)
+
+This directory contains library code for running Zulip
+bots that react to messages sent by users.
+
+This bot will allow you to comment on a GitHub issue.
+You should preface messages with `@comment` or `@gcomment`.
+You will need to have a GitHub account, and a GitHub OAuth token. 
+
+## Setup 
+Before running this bot, make sure to get a GitHub OAuth token.
+The token will need to be authorized for the following scopes: `gist, public_repo, user`.
+Store it in the `github_token.txt` file.
+The `github_token.txt` file should be located at `~/github_token.txt`.
+Please input info like this:
+`/<username>/<repository_owner>/<repository>/<issue_number>/<your_comment`.
+
+
+## Running the bot
+
+Here is an example of running the "githubcomment" bot from
+inside a Zulip repo:
+
+    cd ~/zulip/contrib_bots
+    ./run.py lib/githubcomment.py --config-file ~/.zuliprc-prod
+
+Once the bot code starts running, you will see a
+message explaining how to use the bot, as well as
+some log messages.  You can use the `--quiet` option
```

Comment:
If you use the "logging" module, than the "quiet" option will suppress logging.info messages.  You can still use logging.warn and logging.error for things that you don't think are noise.